mmv1/third_party/terraform/services/storagetransfer/resource_storage_transfer_job_test.go

```diff
@@ -220,6 +220,14 @@ func TestAccStorageTransferJob_transferOptions(t *testing.T) {
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
+			{
+				Config: testAccStorageTransferJob_transferOptions_withTimeCreated(envvar.GetTestProjectFromEnv(), testDataSourceBucketName, testDataSinkName, testTransferJobDescription, true, false, true, testOverwriteWhen[2], testPubSubTopicName, "TIME_CREATED_PRESERVE_AS_CUSTOM_TIME"),
```

Comment:
Can we test the updatability by adding another step that changes some of the new fields' values? To make sure that we can update the fields correctly?